tests/unittests/operators/types_tests.py

```
@@ -132,3 +132,13 @@ def test_pipeline_run_info(self):
         )
         new_obj = types.PipelineRunInfo.from_json(dict_rep)
         self.assertEqual(new_obj, run_info)
+
+
+class TestTextFieldViewType(unittest.TestCase):
+    def test_serialize_multiline(self):
+        view = types.TextFieldView(multiline=True, rows=5)
+
+        dict_rep = view.to_json()
+
+        self.assertTrue(dict_rep["multiline"])
```

Ruff (0.15.7)

[warning] Use a regular `assert` instead of unittest-style `assertTrue`

Replace `assertTrue(...)` with `assert ...`

(PT009)